scripts/globals/job_utils/rune_fencer.lua

```diff
@@ -34,19 +34,28 @@ local function enforceRuneCounts(target)
     local RUNLevel = getRUNLevel(target)
     local maxRunes = RUNLevel >= 65 and 3 or RUNLevel >= 35 and 2 or 1
     local effects = target:getStatusEffects()
-    local runes = {}
+    local oldestRune = nil
+    local oldestRuneDuration = 0
     local i = 0
 
     for _, effect in ipairs(effects) do
         local type = effect:getType()
         if type >= xi.effect.IGNIS and type <= xi.effect.TENEBRAE then
-            runes[i+1] = effect
+
+            local remainingDuration = effect:getTimeRemaining()
+
+            if oldestRune == nil then
+                oldestRune = type
+                oldestRuneDuration = remainingDuration
+            elseif remainingDuration < oldestRuneDuration then
+                oldestRune = type
+                oldestRuneDuration = remainingDuration
+            end
```

Comment:
Can this be collapsed into?:
```lua
            if oldestRune == nil or remainingDuration < oldestRuneDurationthen
                oldestRune = type
                oldestRuneDuration = remainingDuration
            end
```

Reply:
perhaps we shoudl expand `removeOldestManeuver ()` into something less pup specific, or make a version for RUNs JAs and then a lot of script can be reduced

Reply:
Done, and `removeOldestManeuver` could be modified later into a `removeOldestEffectInRange` with input of startEffectID/endEffectID They function identically other than the start/end effect.